metals/src/main/scala/scala/meta/internal/metals/DocumentSymbolProvider.scala

```diff
@@ -105,7 +105,7 @@ class DocumentSymbolProvider(trees: Trees) {
             addChild(s"new $name", SymbolKind.Interface, t.pos, selection, "")
             newOwner()
           } else continue()
-        case _: Source | _: Template | _: Term.Block =>
+        case _: Source | _: Template | _: Term.Block | _: Term.Try =>
```

Comment:
I gave this some thought and I was wondering on whether we should introduce a new owner, similarly to how we do for `new Foo {}`, since it introduces a different scope.

Or maybe this is fine, I don't know.

I implemented this myself the other day and I got stuck on this point 😂

Reply:
I was also thinking about it and I think we should try with new owner.

Reply:
Ok, I changed it around and also added finally and catch. @gabro what do you think?